apps/standalone/src/app/hooks/useStandaloneAppContext.ts

```diff
@@ -39,12 +39,15 @@ const standaloneAppContext: Omit<AppContextProps, 'fetch' | 'metrics'> = {
 export const useStandaloneAppContext = (): AppContextProps => {
   const { username } = React.useContext(AuthContext);
   const fetch = useFetch();
-  const metrics = useMetrics();
+  const getCliArtifacts = useCliArtifacts();
 
   return {
     ...standaloneAppContext,
+    settings: {
+      isRHEM: window.isRHEM || false,
+    },
     user: username,
     fetch,
-    metrics,
+    getCliArtifacts,
```

Comment:
no need to get the function via hook, lets just import it and reference it here
```suggestion
    getCliArtifacts: fetchCliArtifacts,
```